### ISSUE TITLE
Add rendering for landuse=depot

### ISSUE THREAD

**Issue by IgorEliezer** [gravitystorm/openstreetmap-carto#3480]
### Expected behavior
"Depot" (`landuse=depot`) is an area for storage of buses, trucks, wagons, goods and material for logistics/transportation and public service companies, such as garbage trucks, school buses, police vehicles and urban maintenance equipment. Despite being rare, depot areas are usually fairly big.

Rendering `landuse=depot` would be a proper replacement for `landuse=industrial + industrial=depot` which seems to be a workaround to force render bus depots ([example](https://www.openstreetmap.org/way/167869819)) and rail servicing yards ([example](https://www.openstreetmap.org/way/38582438)) while nothing is being assembled or produced.

I suggest using the same pattern of `landuse=garages`, or a lighter version of `landuse=commercial`, with name and no icon.

It may conflict with landuse=railway, since the railway depot can overlap a railway land use ([example](https://www.openstreetmap.org/way/540583094)).

Can be confused with:
* landuse=garages: area for garages next to residential buildings.
* landuse=industrial + industrial=depot: storage area of an industrial plant.
* railway=yard: a node at the center of named freight railway yard, similar to railway=station.

Links:
* Wiki: https://wiki.openstreetmap.org/wiki/Tag:landuse%3Ddepot (status: in use)
* Taginfo: https://taginfo.openstreetmap.org/tags/landuse=depot
* Tag count: 1 016.

### Actual behavior
It's not being rendered at all.

### Links and screenshots illustrating the problem
Examples:
https://www.openstreetmap.org/way/156582306
https://www.openstreetmap.org/way/281352494
https://www.openstreetmap.org/way/41646044
https://www.openstreetmap.org/way/568770554

**Comment by dieterdreist** [comment]
sent from a phone

> On 25. Oct 2018, at 23:14, Igor Eliezer <notifications@github.com> wrote:
> 
> which seems to be a workaround to force render bus depots (example) and rail servicing yards (example) while nothing is being assembled or produced


storage is part of industrial landuse in OSM, so bus depots could fit

**Comment by jeisenbe** [comment]
Rail yards often perform maintenance and repairs on vehicles, and I believe
this happens at most bus depots as well, so they are a form of light
industry.

They are also industrial from a NIMBY standpoint; no one wants a new bus
depot in the back yard of their house or shop.

**Comment by mboeringa** [comment]
> Rendering `landuse=depot` would be a proper replacement for `landuse=industrial + industrial=depot` which seems to be a workaround to force render bus depots ([example](https://www.openstreetmap.org/way/167869819)) and rail servicing yards ([example](https://www.openstreetmap.org/way/38582438)) while nothing is being assembled or produced.

This is not a "workaround", it is a sub-tag and refinement of the landuse=industrial, with a well documented page:

https://wiki.openstreetmap.org/wiki/Key:industrial

The basic question is: do we want to stimulate yet another tag in the landuse key itself, or is the use of the subtag to denote the specific function enough? I am not yet convinced it needs another tag. The argument that "nothing is produced" is weak. Industrial zones often have a wide variety of activity going on. And even "depots" can have serious maintenance going-on on e.g. trains, that borders on the brink of "production". Here in the Netherlands, they almost totally re-build trains during their mid-life update on the depot / maintenance yard...

**Comment by polarbearing** [comment]
> is the use of the subtag to denote the specific function enough?

yes. This tag was already rejected twice, #2989 and #1716.

**Comment by matkoniecz** [comment]
> Whatever the case is about proper tagging, landuse=depot is clearly the favored tag.

I may be highly biased here but I disagree. First of all usage of all this tags is really low, below 1000 landuse=depot started earlier, if you take it into account usage may be considered nearly equal.

**Comment by matkoniecz** [comment]
> rail servicing yards (example) 

This is a clear landuse=railway, railway=depot. I have no idea why landuse=depot would be preferable.

**Comment by matkoniecz** [comment]
I propose to close it (again, see #2989 and #1716) as tag is rarely used and single tag for 

- container depot (`landuse=commercial ` or `landuse=industrial` in some cases)
- bus depot (clear `landuse=commercial `)
- railway depot (clear `landuse=railway `)

just because name is similar makes little to no sense for me. And there should be a really good reason to start promoting a new landuse tag.

See also https://taginfo.openstreetmap.org/keys/landuse#values - this tag is less popular than `landuse=yes`, `landuse=traffic_island`, `landuse=clearing`, `landuse=scrub`, `landuse=winter_sports` etc.

Note that 

>  Landuse=depot (1016 uses), industrial=depot (657 uses), depot=* (479 uses).

comparison is misleading for one fundamental reason - it is not considering many more depots that are mapped as part of standard landuses without `depot=*` tag like for example https://www.openstreetmap.org/way/475661134 https://www.openstreetmap.org/way/419426076  https://www.openstreetmap.org/way/101704683 https://www.openstreetmap.org/way/216723902

**Comment by matkoniecz** [comment]
@Adamant36 Should be done for things mentioned in discussion above.

**Comment by Adamant36** [comment]
@matkoniecz, I agree. So it would be nice if you did it. Since they are your points.

**Comment by matkoniecz** [comment]
In this case "Should be done" was intended to mean "I made edits necessary to document it, hopefully nothing was missed".

**Comment by jeisenbe** [comment]
Thank you for being willing to edit the wiki pages, Mateusz! It’s a
difficult job and usually no one compliments you for doing it.
-Joseph

**Comment by Adamant36** [comment]
@matkoniecz, oh. I miss understood your meaning. Thanks for dealing with the wiki stuff.
